pkg/common/k8s/client.go

```diff
@@ -45,14 +45,21 @@ import (
 	"k8s.io/client-go/transport/spdy"
 
 	"github.com/cilium/cilium/api/v1/models"
-	"github.com/cilium/cilium/cilium-cli/defaults"
 	ciliumv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
 	ciliumv2alpha1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
 	ciliumClientset "github.com/cilium/cilium/pkg/k8s/client/clientset/versioned"
 	"github.com/cilium/cilium/pkg/safeio"
 	"github.com/cilium/cilium/pkg/versioncheck"
 )
 
+// TODO: should we move this into cilium/defaults ?
+// and should all references from cilium-cli point to them as well ?
+const (
+	AgentContainerName                  = "cilium-agent"
+	ClusterMeshBinaryName               = "/usr/bin/clustermesh-apiserver"
+	ClusterMeshKVStoreMeshContainerName = "kvstoremesh"
```

Comment:
Maybe these can go in `pkg/clustermesh/common`.  There's already stuff like logfield constants in there.

Reply:
Gotcha, let me take a look. Probably a good idea to avoid duplicating these if possible.

Reply:
Mmmm after looking at the usage pattern, I am unsure. It seems cilium-cli already declares a lot of clustermesh-related [consts](https://github.com/cilium/cilium/blob/v1.17.0-pre.1/cilium-cli/defaults/defaults.go#L39), and these values seem to be hardcoded in a lot of other places around the repo, meaning having them as consts seems to be more of a helper than wanting to avoid duplication.

This is similar for AgentContainerName, therefore I would be willing to leave these in as-is for now, and maybe revisit consts/defaults as a whole later. wdyt ?